sim/battle.ts

```diff
@@ -2247,7 +2247,7 @@ export class Battle extends Dex.ModdedDex {
 
 		// multi-target modifier (doubles only)
 		if (move.spreadHit) {
-			const spreadModifier = move.spreadModifier || 0.75;
+			const spreadModifier = move.spreadModifier || this.gameType === 'free-for-all' ? 0.5 : 0.75;
```

Comment:
Not actually what the old code did, except I don't know why `move.spreadModifier` exists anyway.

Reply:
Oh is there an order of operations problem here?

Reply:
The old code was effectively `move.spreadModifier ? move.spreadModifier : 0.75` i.e. the value of `move.spreadModifier`, if present, was used instead of `0.75`. The new code treats `move.spreadModifier` as a boolean instead.

Reply:
In other words, yes.

Reply:
`(move.spreadModifier || this.gameType === 'free-for-all') ? 0.5 : 0.75` is how this is interpreted.

Reply:
My bad. As it happens `move.spreadModifer` is never defined, but I imagine if it were it would probably be `0.5` anyway.

Reply:
Sorry, it simply hadn't occurred to me that you wanted `move.spreadModifier || (this.gameType === 'free-for-all' ? 0.5 : 0.75)`. Thanks Zarel for explaining it properly.